commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AddAuditLogHandler.java

```diff
@@ -0,0 +1,48 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.offline.OfflineCommand;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+public final class AddAuditLogHandler implements OnlineCommand, OfflineCommand {
+
+    private final String handlerName;
+
+    public AddAuditLogHandler(String name) {
+        if (name == null) {
+            throw new IllegalArgumentException("Handler name must be specified as non null value");
+        }
+        if (name.isEmpty()) {
+            throw new IllegalArgumentException("Handler name cannot be empty");
+        }
+
+        this.handlerName = name;
+    }
+
+    @Override
+    public void apply(OnlineCommandContext ctx) throws Exception {
+        Operations ops = new Operations(ctx.client);
+        Address handlerAddress = Address.coreService("management")
+                .and("access", "audit")
+                .and("logger", "audit-log")
+                .and("handler", handlerName);
+
+        if (!ops.exists(handlerAddress)) {
```

Comment:
Ah, this is the `AddAuditLogHandler` class. No, I'm not insisting on changing this -- if the class has a more reasonable name, it can stay as is. Or if we decide to change the way how handlers are "enabled".